examples/evaluation/evaluate_with_model_validation.py

```diff
@@ -57,20 +57,17 @@ def double_positive(_, builtin_metrics):
     # mlflow.evaluate() call via model URI.
     baseline_model_uri = mlflow.sklearn.log_model(baseline_model, "baseline_model").model_uri
 
-    try:
-        mlflow.evaluate(
-            candidate_model_uri,
-            eval_data,
-            targets="label",
-            model_type="classifier",
-            dataset_name="adult",
-            evaluators=["default"],
-            validation_thresholds=thresholds,
-            custom_metrics=[double_positive],
-            baseline_model=baseline_model_uri,
-        )
-    except ModelValidationFailedException as e:
-        print("Model failed to pass validation criteria! See detailed error message:")
-        print(e)
-    else:
-        print("Model passed validation criteria!")
+    mlflow.evaluate(
+        candidate_model_uri,
+        eval_data,
+        targets="label",
+        model_type="classifier",
+        dataset_name="adult",
+        evaluators=["default"],
+        validation_thresholds=thresholds,
+        custom_metrics=[double_positive],
+        baseline_model=baseline_model_uri,
+    )
+    # If you would like to catch model validation failures, you can add try except clauses around
+    # the mlflow.evaluate() call and catch the ModelValidationFailedException, imported at the top
+    # of this file.
```

Comment:
Getting rid of catching model validation exceptions so to catch future model validation breaking changes.